### DISCUSSION TITLE
Validating Length of number or float with precision

### DISCUSSION THREAD

**Discussion by bbmmurali** [data-driven-forms/react-forms#1375] [Need help] [answered]
Hi,
  Is there a attribute to validate the length of a number or float? In case of float is there a way to validate the number of digits after decimal without custom function?

**Comment by rvsia** [comment, marked as answer]
Hello, I suggest to use regular expression. (Something like https://stackoverflow.com/questions/4690986/regex-needed-to-match-number-to-exactly-two-decimal-places)
